packages/cozy-konnector-libs/src/helpers/cozy-client-js-stub.js

```diff
@@ -5,6 +5,8 @@ const uuid = require('uuid/v5')
 const sha1 = require('uuid/lib/sha1')
 const bytesToUuid = require('uuid/lib/bytesToUuid')
 const mimetypes = require('mime-types')
+const rootPath = JSON.parse(process.env.COZY_FIELDS).folder_to_save
```

Comment:
this is `undefined` in travis and travis fails when JavaScript wants to `parse(undefined)`.

Reply:
Fixed